src/index.html

```diff
@@ -28,5 +28,6 @@
   </script>
   <script async src='https://www.google-analytics.com/analytics.js'></script>
   <script src="https://ajax.googleapis.com/ajax/libs/hammerjs/2.0.8/hammer.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/web-animations/2.2.5/web-animations.min.js"></script>
```

Comment:
Would it make sense to make this script `async`, think that it would be loaded by the time someone interacts with something animated?